### PR TITLE
Formatting cleanup of kestrel.md and SignalR's javascript-client.md

### PR DESCRIPTION
**Pull request by halter73**

### CODE REVIEW

File: aspnetcore/signalr/javascript-client.md [hunk 1]
```diff
@@ -213,15 +213,16 @@ const connection = new signalR.HubConnectionBuilder()
...
+            if (elapsedMilliseconds < 60000) {
+                // If we've been reconnecting for less than 60 seconds so far,
+                // wait between 0 and 10 seconds before the next reconnect attempt.
+                return Math.random() * 10000;
+            } else {
+                // If we've been reconnecting for more than 60 seconds so far, stop reconnecting.
+                return null;
+            }
+        }
+    })
```
Comment (halter73) [inline aspnetcore/signalr/javascript-client.md]: This line is the main reason for creating this PR. The previous inconsistent indentation made it difficult to see that a `}` was missing.

Comment (scottaddie) [inline aspnetcore/signalr/javascript-client.md]: We terminate JS lines elsewhere in the doc with semi-colons. I suggest we do the same here.

```suggestion
    });
```

Comment (halter73) [reply aspnetcore/signalr/javascript-client.md]: I almost applied this change until I noticed the `.build();` on the next line. I think it's fin as is.

Comment (scottaddie) [reply aspnetcore/signalr/javascript-client.md]: Sorry about that. I obviously missed that line.

**Comment by scottaddie** [comment]
Thank you for the PR, @halter73!

**Pull request merged**